scylladb/scylladb-docs-homepage
# Update landing & theme to 1.7

## Motivation

Closes https://github.com/scylladb/scylladb-docs-homepage/issues/11 and updates the theme to the latest version.

## How to test

1. Build the docs.
2. Open the docs homepage in your browser.
4. Check that the button on the homepage renders correctly, as follows:
   
    ![image](https://github.com/scylladb/scylladb-docs-homepage/assets/9107969/d2b7a5c8-2e77-4319-8031-cf30ad4f4b85)

5. Check that topic boxes on the homepage open in the same tab:

    ![image](https://github.com/scylladb/scylladb-docs-homepage/assets/9107969/e0e29c4b-2d62-4ae1-96d0-0a5512d845aa)

6. Check that the theme version in the footer  is updated to 1.7.1.
